modules/UI/videolayout/VideoLayout.js

```diff
@@ -90,12 +92,13 @@ const VideoLayout = {
     getRemoteVideoType(id) {
         const state = APP.store.getState();
         const participant = getParticipantById(state, id);
+        const isScreenShare = isScreenShareParticipantById(state, id);
 
-        if (participant?.isFakeParticipant) {
+        if (participant?.fakeParticipant && !isScreenShare) {
```

Comment:
`isScreenShare` already includes the `participant?.fakeParticipant` check in the selector it uses